torchvision/prototype/transforms/_color.py

```diff
@@ -120,5 +121,70 @@ def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
 
         for transform in params["image_transforms"]:
             input = transform(input)
+        return input
+
+
+class _RandomChannelShuffle(Transform):
```

Comment:
I made this private for now, but we could also make it public. In that case, we should have a `image_permute_channels` kernel.